redhat-cop/infra-ansible
# updated all documentation to reflect #147 update from @oybed

### What does this PR do?
This is a PR that updates the documentation according to #147. Nothing too fancy, just required some man hours.

### How should this be tested?
No automated testing possible. Just click through the documentation to see how I've updated it to the new standard.

### Is there a relevant Issue open for this?
resolves #147 

### Other Relevant info, PRs, etc.

### People to notify
cc: @oybed
